src/main/java/com/backend/webproject/managers/PaymentManager.java

```diff
@@ -1,31 +1,45 @@
 package com.backend.webproject.managers;
 
+import javax.servlet.http.HttpServletRequest;
+
 import com.backend.webproject.dao.PaymentDAO;
 import com.backend.webproject.entity.Payment;
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.PathVariable;
 
 @Component
 public class PaymentManager {
-    @Autowired
-    PaymentDAO dao;
-
-    public int processPayment(int openCoupon, int shoppingFinalAmount, String paymentStatus, int shoppingCartID){
-        Payment payment = new Payment();
-        payment.setOpenCoupon(openCoupon);
-        payment.setShoppingFinalAmount(shoppingFinalAmount);
-        payment.setPaymentStatus(paymentStatus);
-        payment.setShoppingCartID(shoppingCartID);
-        return dao.processPayment(payment);
-    }
-
-    public Payment searchPaymentByID(int pid){
-        return dao.searchPaymentByID(pid);
-    }
-
-    public Payment searchPaymentByCartID(int cartID){
-        return dao.searchPaymentByCartID(cartID);
-    }
-    
-}
+
+	@Autowired
+	private PaymentDAO paymentJDBC;
+
+	@Autowired
+	private Payment payment;
+
+	public Payment searchByID(@PathVariable("id") int id) {
+		Payment pay = paymentJDBC.searchPaymentByID(id);
+
+		return pay;
+	}
+
+	public Payment searchByCartID(@PathVariable("id") int id) {
+		Payment pay = paymentJDBC.searchPaymentByCartID(id);
+
+		return pay;
+	}
```

Comment:
You can return de function call directly and can erase the PathVariable annotaion.